packages/lumx-react/src/components/index.ts

```diff
@@ -34,7 +34,6 @@ export const ColorPalette = {
     grey: 'grey',
 } as const;
 export type ColorPalette = ValueOf<typeof ColorPalette>;
-export type Color = ColorPalette | string;
```

Comment:
I think we should keep the `Color` type as an alias to the `ColorPalette` so that users do not have to replace it from their import.